packages/opentelemetry-plugin-http/src/http.ts

```diff
@@ -312,7 +312,7 @@ export class HttpPlugin extends BasePlugin<Http> {
 
       return context.with(propagation.extract(headers), () => {
         const span = plugin._startHttpSpan(
-          `${method} ${pathname}`,
+          `HTTP ${method}`,
           spanOptions
         );
```

Comment:
The linter wants this to be a single line call now that it is shorter

```typescript
        const span = plugin._startHttpSpan(`HTTP ${method}`, spanOptions);
```

Reply:
got it, thx

Reply:
you probably need to run `npm run lint:fix` and it will lint it for you

Reply:
Already done, I have really forgot about it :)